src/core/appgateway.tf

```diff
@@ -735,6 +752,20 @@ data "azurerm_key_vault_certificate" "app_gw_api_app" {
   key_vault_id = module.key_vault.id
 }
 
+###
+# kv where the certificate for api-web domain is located
+###
+data "azurerm_key_vault" "profile_kv" {
+  name                = format("%s-profile-kv", local.project)
+  resource_group_name = format("%s-profile-sec-rg", local.project)
+}
+
+data "azurerm_key_vault_certificate" "app_gw_api_web" {
+  name         = var.app_gateway_api_web_certificate_name
+  key_vault_id = data.azurerm_key_vault.profile_kv.id
+}
```

Comment:
Are you sure that the SSL certificate is into profile key vault? Generally is in the key vault `kv` or `common`

Reply:
this has to be changed after https://github.com/pagopa/io-azure-devops/pull/22, where we put the certificate in the correct kv

Reply:
fixed in https://github.com/pagopa/io-infra/pull/655/commits/eed4c9b642725fd30debee6e587eb753863a83a5